apps/apps_container.cpp

```diff
@@ -411,7 +416,7 @@ void AppsContainer::redrawWindow(bool force) {
 
 void AppsContainer::activateExamMode(GlobalPreferences::ExamMode examMode) {
   assert(examMode != GlobalPreferences::ExamMode::Off && examMode != GlobalPreferences::ExamMode::Unknown);
-  reset();
+  reset(examMode != GlobalPreferences::ExamMode::Standard);
```

Comment:
Why did you remove the files if the exam mode isn't standard ?

Reply:
Because the standard mode is like in France and I don't know what are the modalities for the others countries . Maybe others countries want that the files are deleted in the exam mode ^^

Reply:
OK